app/assets/stylesheets/leaderboard.css

```diff
@@ -58,32 +72,74 @@
 }
 
 .platform-column{
-  width: calc(150 * var(--fpx));
+  width: calc(84 * var(--fpx));
 }
 
 .location-column{
-  width: calc(120 * var(--fpx));
+  width: calc(84 * var(--fpx));
+}
+
+.leaderboard-item.location-column{
+  padding-left: calc(24 * var(--fpx));
+}
+
+.leaderboard-item.platform-column::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: calc(18 * var(--fpx));
+  background-image: var(--ico-url);
+  background-size: auto 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  transition: background-image 360ms ease-out;
+}
+
+.leaderboard-item-wrapper:hover .platform-column::before {
+  background-image: var(--ico-url-hover);
+  transition: background-image 72ms ease-out;
+}
+
+.location-flag {
+  position: absolute;
+  top: 0;
+  left: calc(50% - 9 * var(--fpx));
+  width: calc(18 * var(--fpx));
+  height: calc(18 * var(--fpx));
```

Comment:
Minor comment - it might make sense to have the platform and flag icons for ranks 1-3 bigger than for the other ranks, as the text is bigger for the top ranks

Reply:
Good idea. I've updated the style.

Reply:
Yeeee that looks nice :)